plugin/pkg/shoot/validator/admission.go

```diff
@@ -876,21 +870,13 @@ func (c *validationContext) validateShootNetworks(a admission.Attributes, worker
 
 	if c.seed != nil {
 		if c.shoot.Spec.Networking.Pods == nil && !workerless {
-			if c.seed.Spec.Networks.ShootDefaults != nil {
-				if cidrMatchesIPFamily(*c.seed.Spec.Networks.ShootDefaults.Pods, c.shoot.Spec.Networking.IPFamilies) {
-					c.shoot.Spec.Networking.Pods = c.seed.Spec.Networks.ShootDefaults.Pods
-				}
-			} else if slices.Contains(c.shoot.Spec.Networking.IPFamilies, core.IPFamilyIPv4) {
+			if slices.Contains(c.shoot.Spec.Networking.IPFamilies, core.IPFamilyIPv4) {
 				allErrs = append(allErrs, field.Required(path.Child("pods"), "pods is required"))
 			}
 		}
 
 		if c.shoot.Spec.Networking.Services == nil {
-			if c.seed.Spec.Networks.ShootDefaults != nil {
-				if cidrMatchesIPFamily(*c.seed.Spec.Networks.ShootDefaults.Services, c.shoot.Spec.Networking.IPFamilies) {
-					c.shoot.Spec.Networking.Services = c.seed.Spec.Networks.ShootDefaults.Services
-				}
-			} else if slices.Contains(c.shoot.Spec.Networking.IPFamilies, core.IPFamilyIPv4) {
+			if slices.Contains(c.shoot.Spec.Networking.IPFamilies, core.IPFamilyIPv4) {
 				allErrs = append(allErrs, field.Required(path.Child("services"), "services is required"))
 			}
 		}
```

Comment:
these ifs can be combined.
```suggestion
if c.shoot.Spec.Networking.Services == nil && slices.Contains(c.shoot.Spec.Networking.IPFamilies, core.IPFamilyIPv4) {
    allErrs = append(allErrs, field.Required(path.Child("services"), "services is required"))
}
```